docs/source/using-discovery/slurm.md

```diff
@@ -0,0 +1,769 @@
+gpu          up    8:00:00      3  alloc d[1002,1015,1019]
+gpu          up    8:00:00      4   idle c[2180-2183]
+The current `TimeLimit` for the queues:
+sinfo  -o "%12P %.10A %.11l"
+PARTITION    NODES(A/I)   TIMELIMIT
+debug           402/174       20:00
+express         403/180     1:00:00
+short*          401/178  1-00:00:00
+long             224/47  5-00:00:00
+large           376/172     6:00:00
+gpu               41/17     8:00:00
+multigpu          41/17  1-00:00:00
+lowpriority     118/102  1-00:00:00
+reservation     617/402 100-00:00:0
+ai-jumpstart       2/15  2-00:00:00
+allshouse           5/7    infinite
+bansil             15/4 30-00:00:00
+ce-mri             3/10 30-00:00:00
+chen               0/12 30-00:00:00
+ctbp               0/20 30-00:00:00
+.
+.
+.
+:::
+
+View information about a specific partition (e.g., `short`, `gpu`, `long`):
+
+:::{code} bash
+sinfo -p <partition_name>
+:::
+
+Or, only view nodes in a certain state:
+
+:::{code} bash
+sinfo -p <partition> -t <state>
+:::
+
+You can use the `--Format` flag to get more information or a specific format for the output:
+
+:::{code} bash
+sinfo -p <partition> -t idle --Format=gres,nodes
+:::
+
+Below command will show detailed information about all nodes in the cluster, including the node name, state, CPU architecture, memory, and available features:
+
+:::{code} bash
+sinfo -N -l
+:::
+
+View what features a node has:
+
+:::{code} bash
+sinfo -n <node> --Format=nodes,nodelist,statecompact,features
+View what nodes are in what state in a partition using `statecompact`:
+sinfo -p <partition> --Format=time,nodes,statecompact,features,memory,cpus,nodelist
+:::
+
+### Using scontrol
+The `scontrol` command is a command-line utility that allows users to view and control Slurm jobs and job-related resources. It provides a way to check the status of jobs, modify job properties, and perform other job-related tasks.
+
+Some of the tasks that can done using `scontrol` include:
+
+Viewing job status and properties: `scontrol` can display detailed information about a job, including its status, node allocation, and other properties.
+
+Modifying job properties: `scontrol` allows users to modify job properties such as the job name, the number of nodes, the time limit, and other parameters.
+
+Managing job dependencies: `scontrol` provides a way to specify job dependencies and view the dependencies between jobs.
+
+Suspending and resuming jobs: `scontrol` can stop and resume running jobs, allowing users to temporarily halt jobs or continue them as needed.
+
+Canceling jobs: `scontrol` can cancel jobs that are running or queued, allowing users to stop jobs that are no longer needed.
+
+Overall, `scontrol` is a powerful tool for managing Slurm jobs and job-related resources. Its command-line interface allows users to perform a wide range of tasks, from checking the status of jobs to modifying job properties and managing dependencies.
+
+### Controlling jobs
```

Comment:
I am not sure if we should cut this section since the information is above or if we should include it in both locations.

Reply:
@joseph-cooke, nice catch! I merged the two sections!